config.py

```diff
@@ -17,9 +17,9 @@
 BOT_ASYNC = True
 BOT_ASYNC_POOLSIZE = 10
 BOT_IDENTITY = {'token': secrets.get('Slackbot', 'token')}
-BOT_ADMINS = ('@mattreed79', '@paulkarayan')
+BOT_ADMINS = ('@paul')
 BOT_PREFIX = '!'
-BOT_ALT_PREFIXES = ('@bender',)
+BOT_ALT_PREFIXES = ('@bender','@jiraed')
```

Comment:
This is interesting I wonder if we can use this as the command rather than the summoning of a bot by name.

So it could be `@jira <something>` or even `!jira <something>` and so on.